server/channel_player.go

```diff
@@ -688,3 +688,26 @@ func (server ChannelServer) playerBumpDamage(conn mnet.Client, reader mpacket.Re
 	plr.DamagePlayer(int16(damage))
 
 }
+
+func (server *ChannelServer) getPlayerInstance(conn mnet.Client, reader mpacket.Reader) (*field.Instance, error) {
```

Comment:
This should probably be `(server ChannelServer)` as this function shouldn't modify ChannelServer